playbook/app/pb_kits/playbook/pb_table/docs/_table_header.html.erb

```diff
@@ -1,58 +1,69 @@
-<%= pb_rails("table", props: { size: "lg"} ) do %>
+<% data_rows = [
```

Comment:
I think it might be good to add a comment saying this is just example data for demonstration purposes.